examples/health-checks.yml.advanced

```diff
@@ -0,0 +1,19 @@
+tcp:
```

Comment:
Consider renaming these files with the extension at the end (e.g., `health-checks.advanced.yml`) so they get proper syntax highlighting.